static/css/jira-configuration-new.css

```diff
@@ -88,14 +88,12 @@
 }
 
 .error-summary-errors-header {
-    font-size: 12px;
-    color: #6b778c;
+    padding-top: 1rem;
```

Comment:
rem is typically used for font-size (relative to the root measurement). Use em for padding/margins so the size adjusts as the screen does.